README.md

```diff
@@ -224,3 +175,35 @@ jobs:
           develocity-url: 'https://<MY_DEVELOCITY_URL>'
           develocity-access-key: ${{ secrets.<DEVELOCITY_ACCESS_KEY> }}
 ```
+
+_Note:_<br>
+Some parameters need to be adjusted here:
+- The workflow name (here `PR Build`) triggered when a pull-request is submitted
+- The Develocity URL (here `https://<MY_DEVELOCITY_URL>`)
+- The secret name holding the Develocity access key (here `<DEVELOCITY_ACCESS_KEY>`)
+ 
+#### Implementation details
+
+The action will download any saved Build Scan® and publish them to Develocity.
+
+The list of pull-request authors allowed to publish a Build Scan® can be specified by the csv parameter `authorized-users-list`. 
```

Comment:
What is the behavior if this list is missing? Are no scans published or are all scans published?

Reply:
All scans would be published, I changed the comment to:
> The action will publish Build Scans® if the initial pull-request author belongs to the list or if the list is empty.